dotCMS/src/main/java/com/dotcms/rest/IntegrityResource.java

```diff
@@ -1010,24 +1032,19 @@ public Response discardConflicts ( @Context final HttpServletRequest request, @C
     @Path("/fixconflictsfromremote/{params:.*}")
     @Consumes(MediaType.MULTIPART_FORM_DATA)
     @Produces("text/plain")
-    public Response fixConflictsFromRemote ( @Context final HttpServletRequest request,
-                                             @FormDataParam("DATA_TO_FIX") InputStream dataToFix, @FormDataParam("AUTH_TOKEN") String auth_token_digest,
-                                             @FormDataParam("TYPE") String type ) throws JSONException {
-
-        String remoteIP = null;
+    public Response fixConflictsFromRemote (@Context final HttpServletRequest request,
+                                            @FormDataParam("DATA_TO_FIX") final InputStream dataToFix,
+                                            @FormDataParam("AUTH_TOKEN") final String auth_token_digest,
+                                            @FormDataParam("TYPE") final String type ) throws JSONException {
+        final String remoteIp = resolveRemoteIp(request);;
```

Comment:
Codacy found an issue: [An empty statement (semicolon) not part of a loop](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6151243)